app/views/research_sessions/questions/researcher.html.erb

```diff
@@ -2,6 +2,11 @@
 <div class="main-content">
   <%= render 'shared/error_summary' %>
   <h1 class="page-heading">Researcher</h1>
+
+  <div class="research-guidance">
+    <p>The researcher’s details will go directly on the sheet sent to participants of this session so they can contact you with any queries. The researcher should refer to the safeguarding code of conduct in <a href="http://b-hive.barnardos.org.uk/Interact/Pages/Content/Document.aspx?id=5100&SearchId=">BREC Guidance</a> so they are confident they know what to do if a safeguarding concern arises during the session.</p>
```

Comment:
If b-hive supports https then I'd favour linking to that version. (I can't test from outside the network)